src/app/signin/page.tsx

```diff
@@ -26,6 +28,13 @@ const Signin = () => {
 
     if (!res?.error) {
       router.push('/');
+    } else {
+      toast('Error Signing in', {
```

Comment:
Can't this error be more specific? res?.error probably. Might be good to display the returned error insted of static message.

Reply:
Yeah should address this in a follow up, merging for now as a v1